libc/src/__support/CPP/bit.h

```diff
@@ -238,13 +238,36 @@ LIBC_INLINE constexpr To bit_or_static_cast(const From &from) {
   }
 }
 
+#define ADD_SPECIALIZATION(NAME, TYPE, BUILTIN)                                \
```

Comment:
Can we have different macro names to distinguish between this `ADD_SPECIALIZATION` and the previous `ADD_SPECIALIZATION`?  It is going to be a bit tricky for readability to see which definition apply to which `ADD_SPECIALIZATION` when searching.

Reply:
done in cea5f1fe0cef PTAL